libc-top-half/musl/src/misc/dl.c

```diff
@@ -0,0 +1,31 @@
+#include <stdint.h>
```

Comment:
Perhaps add a comment here about why this file exists?  

Mention that it used to build libdl.so and that the intent is to then link with non-stub versions at runtime?